src/ngeo.js

```diff
@@ -43,4 +43,4 @@ goog.require('ol.style.AtlasManager'); // nowebpack
  * The default template base URL for modules, used as-is by the template cache.
  * @type {string}
  */
-ngeo.baseModuleTemplateUrl = 'ngeo';
+ngeo.baseModuleTemplateUrl = 'ngeo'; // nowebpack
```

Comment:
I think you can delete the whole file and remove all the `// nowebpack` here

Reply:
And that mean, that you can also remove all the "goog.require('ngeo')" in component, on webpack